web/plugin-manager/src/pages/Manager.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState } from "react";
+import PluginBlock from "../components/pluginBlock";
+import { Plugin, PluginProps } from "../../../shared/interfaces/IPlugin";
+import massaLogoLight from "../assets/MASSA_LIGHT_Detailed.png";
+import axiosServices from "../services/axios";
+import { AxiosResponse } from "axios";
+import alertHelper from "../helpers/alertHelpers";
+import { PuffLoader } from "react-spinners";
+function Manager() {
+    let pluginsInfos: AxiosResponse<any, any> = {} as AxiosResponse<any, any>;
+
+    //State to store error
+    const [error, setError] = useState(<></>);
+    //State to store plugins populated
+    const [pluginsPopulated, setpluginsPopulated] = useState([<PuffLoader/>])
+    //Callback to remove Error
+    function removeError(): void {
+        setError(<></>);
+    }
+    function setErrorHandler(errorType: string, errorMessage: string): void {
+        setError(alertHelper(errorType, errorMessage, removeError));
+        setInterval(() => {
+            setError(<></>);
```

Comment:
why not calling removeError ?

Reply:
because it's a setState convention react

Reply:
I don't understand.